### PR TITLE
fix: add missing `kmanila_` prefix for configuration keys

### PR DESCRIPTION
**Pull request by chisacam**
- resolves https://github.com/lablup/backend.ai-ossca-2023/issues/18
- i'm not sure about i changed all of names as it need, please review it and told me it is require changes or not, or need more changes. i'll check and more edit or revert it.
- change default etcd port to i heard correct number.

### CODE REVIEW

**Review by fregataa — APPROVED** [review]
Thanks!
Plus, it will be nice to update `install-dev.sh` to replace etcd port in `storage-proxy.toml` from 8121(or 2379) into `ETCD_PORT`.
You can use `sed_inplace`.

**Review by fregataa — CHANGES_REQUESTED** [review]
please add a news fragment file for timeline checker!

File: configs/storage-proxy/sample.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [etcd]
 namespace = "local"                         # env: BACKEND_NAMESPACE
-addr = { host = "127.0.0.1", port = 2379 }  # env: BACKEND_ETCD_ADDR (host:port)
+addr = { host = "127.0.0.1", port = 8121 }  # env: BACKEND_ETCD_ADDR (host:port)
```
Comment (kyujin-cho) [inline configs/storage-proxy/sample.toml]: `2379` is de-facto standard for Etcd server's listening port number. Please just leave it as-is and let's create a separate issue to create a storage-proxy version of `halfstack.toml` config file.

**Review by kyujin-cho — CHANGES_REQUESTED** [review]

File: configs/storage-proxy/sample.toml (hunk 1 above)
Comment (chisacam) [reply configs/storage-proxy/sample.toml]: sure, i'll revert it..
how about changes in `install-dev.sh`? am i revert that too?

Comment (kyujin-cho) [reply configs/storage-proxy/sample.toml]: Yes please.

**Comment by kyujin-cho** [comment]
Closing as we made a decision to lift up kmanila integration from the monorepo and store it as a separated plugin repository.

**Pull request closed without merging**